docbook/reference/en/en-US/modules/Multipart.xml

```diff
@@ -472,6 +472,120 @@ public class CustomerPortalServer {
     }
 }
 </programlisting>
+       <para>In addition to the XML data format, you can also use JSON formatted data to represent your POJO classes.
+           To achieve this goal, you need to plug in a JSON provider into your project. For example,
+           you can add RESTEasy Jackson2 Provider into your project's dependency scope:
+       </para>
+<programlisting><![CDATA[
+<dependency>
+    <groupId>org.jboss.resteasy</groupId>
+    <artifactId>resteasy-jackson-provider</artifactId>
```

Comment:
This is the Jackson, not Jackson2 provider that is mentioned above. So I guess you should have resteasy-jackson2-provider instead.

Reply:
I have fixed this typo. Thanks for checking!